x-pack/plugins/ml/public/application/components/data_grid/data_grid.tsx

```diff
@@ -162,22 +175,44 @@ export const DataGrid: FC<Props> = memo(
             <EuiSpacer size="m" />
           </div>
         )}
-        <EuiDataGrid
-          aria-label={isWithHeader(props) ? props.title : ''}
-          columns={columns}
-          columnVisibility={{ visibleColumns, setVisibleColumns }}
-          gridStyle={euiDataGridStyle}
-          rowCount={rowCount}
-          renderCellValue={renderCellValue}
-          sorting={{ columns: sortingColumns, onSort }}
-          toolbarVisibility={euiDataGridToolbarSettings}
-          pagination={{
-            ...pagination,
-            pageSizeOptions: [5, 10, 25],
-            onChangeItemsPerPage,
-            onChangePage,
-          }}
-        />
+        <div className="mlDataGrid">
+          <EuiDataGrid
+            aria-label={isWithHeader(props) ? props.title : ''}
+            columns={columnsWithCharts.map((c) => {
+              c.initialWidth = 150;
+              return c;
+            })}
+            columnVisibility={{ visibleColumns, setVisibleColumns }}
+            gridStyle={euiDataGridStyle}
+            rowCount={rowCount}
+            renderCellValue={renderCellValue}
+            sorting={{ columns: sortingColumns, onSort }}
+            toolbarVisibility={{
+              ...euiDataGridToolbarSettings,
+              additionalControls: (
+                <EuiButtonEmpty
+                  className={`euiDataGrid__controlBtn${
+                    chartsVisible ? ' euiDataGrid__controlBtn--active' : ''
+                  }`}
+                  size="xs"
+                  iconType="visBarVertical"
+                  color="text"
+                  onClick={toggleChartVisibility}
+                >
+                  {i18n.translate('xpack.ml.dataGrid.histogramButtonText', {
+                    defaultMessage: 'Histogram Charts',
```

Comment:
This should be sentence case 'Histogram charts'.

Reply:
Fixed in 813065d.